nixos/lib/make-disk-image.nix

```diff
@@ -70,6 +82,13 @@
 , # Disk image format, one of qcow2, qcow2-compressed, vdi, vpc, raw.
   format ? "raw"
 
+  # Whether to fix partitions GUIDs/UUIDs to arbitrary values.
+  # Also, to fix last time checked of the ext4 partition if fsType = ext4.
+, deterministic ? true
+
+  # GUID used for root partition
```

Comment:
Is this a *Partition Unique Identifier* for use with `deterministic`?

Reply:
Clarified.